extensions/rvadapter/src/main/java/com/google/android/agera/rvadapter/RepositoryAdapter.java

```diff
@@ -128,7 +133,127 @@ public RepositoryAdapter build() {
       return new RepositoryAdapter(this);
     }
 
+    /**
+     * Builds the {@link RepositoryAdapter} that presents the provided repositories in order and
+     * observes the repositories as well as any additional observables while the provided
+     * {@link Activity} is resumed (between onResumed/onPaused).
```

Comment:
nit: `between {@link Activity#onResume()} and {@link Activity#onPause()}.` ?

Same for the next one, onStart() and onStop().